tests/rbd/rbd_utils.py

```diff
@@ -121,6 +121,64 @@ def check_pool_exists(self, pool_name: str) -> bool:
         log.info(f"pool {pool_name} does not exist on cluster")
         return False
 
+    def create_file_to_import(self, filename="dummy"):
+        """
+        Creates a dummy file on client node to import
+        Args:
+            filename: name of the dummy file
+        Returns:  True -> pass, False -> fail
+        """
+        self.exec_cmd(
+            cmd="dd if=/dev/urandom of={} bs=4 count=5M".format(filename),
+            long_running=True,
+        )
```

Comment:
you could also refactor like this. 

```
cmd = f"dd if=/dev/urandom of={filename} bs=4 count=5M"
if not self.exec_cmd(
    cmd=cmd,
    long_running=True,
):
    raise CreateFileImportError("file import failed")
```            
it could be similar for the following functions too.

Reply:
Sure
Is it for 1. some optimization best practice or 2. a code standard that we want to follow in this repo ?

Reply:
I don't think this is a coding standard, I think this is a small optimization. The only standard I see in my comment is making use of `format strings`.